ehitchhiking-ui/src/app/shared/components/trip/trip.component.html

```diff
@@ -1,7 +1,10 @@
-<i class="position-car material-icons">
+<i class="position-role material-icons" *ngIf="trip.role === userState.Driver" title = "driver">
```

Comment:
bad naming: position-role. Pls rename to user-role-icon

Reply:
okay